src/QemuRunner.jl

```diff
@@ -250,7 +250,7 @@ function run_interactive(qr::QemuRunner, cmd::Cmd; stdin = nothing, stdout = not
     end
     
     # Qemu appears to mess with our terminal
-    Base.reseteof(STDIN)
+    Base.reseteof(stdin)
```

Comment:
`Base.stdin` is shadowed by the `stdin` parameter being passed into this function; so I think we need to make this explicitly `Compat.stdin`.

Reply:
done